src/mmw/templates/registration/activation_complete.html

```diff
@@ -7,7 +7,7 @@
 <p>
     {% trans "Your account is now activated." %}
     {% if not user.is_authenticated %}
-    {% trans "You can log in." %}
+    {% trans "If you had work in progress, please switch back to that tab and login to save your work." %}
```

Comment:
I'd love to do something like stash this in local storage and then rebuild the page instead of expecting users not to mess up their tabs. It seems suboptimal to me that I can't complete any process from one window. Not an issue immediately but I think we should consider dealing with this in a better way later. As it stands this feels like the bank telling me not to use the back button.